### DISCUSSION TITLE
Stuck on Factory Chinese Boot Screen

### DISCUSSION THREAD

**Discussion by IIAIronWolf** [seaky/nspanel_pro_tools_apk#129] [Q&A]
My NSPanel Pro is stuck on the Chinese factory boot screen on startup. It has the QR code mentioned in the 'get out from rabbit hole' section of the roottool documentation: [Get out from rabbit hole](https://github.com/seaky/nspanel_pro_roottool_apk?tab=readme-ov-file#get-out-from-rabbit-hole)
The documentation makes it seem like you can get to a recovery area even if this screen (lost ewelink license) screen is present, since the section ends with "Do the [Get out from rabbit hole](https://github.com/seaky/nspanel_pro_roottool_apk?tab=readme-ov-file#get-out-from-rabbit-hole)".

Has anyone successfully fixed this? I have not found a single thing online that actually shows how to fix this problem. I'm afraid my device is permanently bricked.

**Comment by seaky** [comment]
Hi 
If you stucked on Chinese factory screen just follow my instructions and you will get back your device!

**Reply by IIAIronWolf** [reply]
> Hi 
> 
> If you stucked on Chinese factory screen just follow my instructions and you will get back your device!

Unfortunately, that is why I started this discussion. That does not work.

**Comment by seaky** [comment]
Did you manage to solve this?

**Reply by IIAIronWolf** [reply]
I did not :/

**Reply by gajda** [reply]
> Did you manage to solve this?

Yes, I did manage to get of that chinese factory screen with no license warning. I followed mentioned walkthrough. I had to use external power and special usb cable (OTG). Unfortunately I did something to touch layer of the screen so touch display works just in few areas.

**Reply by IIAIronWolf** [reply]
> > Did you manage to solve this?
> 
> 
> 
> Yes, I did manage to get of that chinese factory screen with no license warning. I followed mentioned walkthrough. I had to use external power and special usb cable (OTG). Unfortunately I did something to touch layer of the screen so touch display works just in few areas.

I believe they were asking me, since this is my Q&A haha. I have not resolved my issue. Still no access to device.

**Reply by seaky** [reply]
> > > Did you manage to solve this?
> > 
> > 
> > 
> > Yes, I did manage to get of that chinese factory screen with no license warning. I followed mentioned walkthrough. I had to use external power and special usb cable (OTG). Unfortunately I did something to touch layer of the screen so touch display works just in few areas.
> 
> I believe they were asking me, since this is my Q&A haha. I have not resolved my issue. Still no access to device.

Ok step back, as I understood you stucked in the chinese factory screen if you stucked on there (btw it is still the ewelink app just another fallback page in the app, if licence data is not available the ewelink app shows this screen so technically you did not brick your device) you should do what I wrote on the RootTool page

Required tools to get the control back:
- Buy an OTG-USB cable, the key is the OTG with regular usb cable it will not work.
- Then you need an extrnal 5VDC power source.

In the nspanel pro roottool page search for the "Get out the rabbit hole" section it fully describes every steps how to recover your device. (how to connect power source how to connect usb etc)
